app/controllers/staff/grids/bulk_time_slots_controller.rb

```diff
@@ -27,7 +27,7 @@ def create
     @schedule = Schedule.new(current_event)
 
     respond_to do |format|
-      format.json { render json: { slots: current_event.time_slots }, status: :ok }
+      fuormat.json { render json: { slots: current_event.time_slots }, status: :ok }
```

Comment:
@YayFiber Looks like a stray 'u' got in here. I wonder if this is being used since this should blow up?

Reply:
Godo eye, will remove. It definitely is used, but I must not have used it since the typo.